### PR TITLE
Added a note to /keys and /.well-known endpoints that the data returned is eventually consistent

### PR DESCRIPTION
**Pull request by larsjohansen-okta**
## Description:
- **What's changed?** 
Added a note to /keys and /.well-known endpoints that the data returned is eventually consistent.
Details for the product change: https://oktainc.atlassian.net/browse/OKTA-228562
- **Is this PR related to a Monolith release?** 2019.05.03

### Resolves:

* [OKTA-229355](https://oktainc.atlassian.net/browse/OKTA-229355)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/api/resources/oidc/index.md [hunk 1]
```diff
@@ -505,6 +505,8 @@ These keys can be used to locally validate JWTs returned by Okta. Standard open-
 
 >Okta also recommends caching or persisting these keys to improve performance. If you cache signing keys and automatic key rotation is enabled, be aware that verification fails when Okta rotates the keys automatically. Clients that cache keys should periodically check the JWKS for updated signing keys.
 
+> Note! The information returned at this endpoint is eventually consistent with the org's configuration, so it could be returning stale information.
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: `Note:` throughout, please.

File: packages/@okta/vuepress-site/docs/api/resources/oidc/index.md [hunk 2]
```diff
@@ -505,6 +505,8 @@ These keys can be used to locally validate JWTs returned by Okta. Standard open-
 
 >Okta also recommends caching or persisting these keys to improve performance. If you cache signing keys and automatic key rotation is enabled, be aware that verification fails when Okta rotates the keys automatically. Clients that cache keys should periodically check the JWKS for updated signing keys.
 
+> Note: The information returned at this endpoint could lag slightly, but will eventually be up-to-date.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: > Note: The information returned to this endpoint could lag slightly, but will eventually be up-to-date.

File: packages/@okta/vuepress-site/docs/api/resources/oidc/index.md [hunk 3]
```diff
@@ -635,6 +637,8 @@ Returns OAuth 2.0 metadata related to your custom authorization server. This inf
 
 > This API doesn't require any authentication.
 
+> Note: The information returned at this endpoint could lag slightly, but will eventually be up-to-date.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: > Note: The information returned to this endpoint could lag slightly, but will eventually be up-to-date.

File: packages/@okta/vuepress-site/docs/api/resources/oidc/index.md [hunk 4]
```diff
@@ -770,6 +774,8 @@ Returns OpenID Connect metadata about your authorization server. This informatio
 
 This API doesn't require any authentication.
 
+> Note: The information returned at this endpoint could lag slightly, but will eventually be up-to-date.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: > Note: The information returned to this endpoint could lag slightly, but will eventually be up-to-date.

File: packages/@okta/vuepress-site/docs/api/resources/oidc/index.md (hunk 2 above)
Comment (jakubvul) [reply packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: `from this endpoint`?

Comment (larsjohansen-okta) [reply packages/@okta/vuepress-site/docs/api/resources/oidc/index.md]: yep, from

**Pull request merged**